core/src/main/resources/SpringApplicationContext.xml

```diff
@@ -17,6 +17,12 @@
 
 	<import resource="springUnmanagedDeployment.xml"/>
 
+	<bean name="applicationEventMulticaster" class="org.springframework.context.event.SimpleApplicationEventMulticaster" autowire="byName">
+		<property name="errorHandler">
```

Comment:
Spring will create this bean if it's not defined, but without an error-handler or task-executor.

Creating our own bean we can control these properties and errors from event handlers won't be propagated back to the event sources, but instead flow to the error-handler.

Reply:
CO0l